subprojects/docs/src/docs/release/notes.md

```diff
@@ -10,6 +10,12 @@ Add-->
 ### Example new and noteworthy
 -->
 
+### Stricter validation with validateTaskProperties
```

Comment:
We should either mention the task name (`validateTaskProperties`) or the task type (`ValidateTaskProperties`) consistently I think. The different casing is confusing.

Reply:
Should also escape the name with backticks (``).